src/engine/menus/unknown.asm

```diff
@@ -50,15 +50,15 @@ Func_18661: ; unreferenced
 
 ; b button
 	ld a, -1
-	call Func_190fb
+	call $50fb ; non-existent routine, points to middle of SetScreenForDuelAnimation
```

Comment:
Hah, fascinating. Surely, this must be a reference to `PlaySFXConfirmOrCancel` (02:50fb) but referenced from the wrong bank (6).

```suggestion
	call PlaySFXConfirmOrCancel ; bug: should be farcall
```

Reply:
Good catch, though they probably meant to use PlaySFXConfirmOrCancel_Bank6, which is in that bank, I'll put a comment